update.sh

```diff
@@ -2,22 +2,29 @@
 set -e
 SEMVER_REGEX="^(0|[1-9][0-9]*)\\.(0|[1-9][0-9]*)\\.(0|[1-9][0-9]*)$"
 
-stable_online_version=$(curl 'https://factorio.com/api/latest-releases' | jq '.stable.headless' -r)
-experimental_online_version=$(curl 'https://factorio.com/api/latest-releases' | jq '.experimental.headless' -r)
-stable_sha256=$(curl "https://factorio.com/download/sha256sums/" | grep "factorio_headless_x64_${stable_online_version}.tar.xz" | awk '{print $1}')
-experimental_sha256=$(curl "https://factorio.com/download/sha256sums/" | grep "factorio_headless_x64_${experimental_online_version}.tar.xz" | awk '{print $1}')
-stable_current_version=$(jq 'with_entries(select(contains({value:{tags:["stable"]}}))) | keys | .[0]' buildinfo.json -r)
-latest_current_version=$(jq 'with_entries(select(contains({value:{tags:["latest"]}}))) | keys | .[0]' buildinfo.json -r)
+# Fetch online stable and experimental versions
+stable_online_version=$(curl -s 'https://factorio.com/api/latest-releases' | jq '.stable.headless' -r)
+experimental_online_version=$(curl -s 'https://factorio.com/api/latest-releases' | jq '.experimental.headless' -r)
+
+# Fetch the binaries
+stable_sha256=$(curl -s "https://factorio.com/get-download/${stable_online_version}/headless/linux64" | grep "factorio_headless_x64_${stable_online_version}.tar.xz" | awk '{print $1}')
+stable_sha256=$(curl -s "https://factorio.com/download/sha256sums/" | grep -E"(factorio_headless_x64|factorio-headless_linux)_${experimental_online_version}.tar.xz" | awk '{print $1}')
```

Comment:
Previous review accidentally removed the experimental SHA256 checksum:
```suggestion
experimental_sha256=$(curl -s "https://factorio.com/download/sha256sums/" | grep -E"(factorio_headless_x64|factorio-headless_linux)_${experimental_online_version}.tar.xz" | awk '{print $1}')
```

Reply:
My bad on that 😅